src/CSV_Parser.php

```diff
@@ -363,4 +425,17 @@ private function _makeRows() {
 		$this->setRows($rows);
 		return $this;
 	}
+
+	/**
+	 * Validate character encoding
+	 *
+	 * @return boolean
+	 */
+	private function _isValidEncoding($encoding = null) {
+	    if (in_array($encoding, mb_list_encodings())) {
```

Comment:
Just read my comments again and that did not make sense! Sorry about that. Meant to say that:

> This method `in_array()` would just return boolean..